multiply.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sat Jul  4 23:15:25 2020
+
+@author: Dell
+"""
+
+
+def multiply(numbers):
+    if numbers == []:
```

Comment:
let's change this for "if not numbers:", since it's the pythonic way to check if a list is empty